components/multiselect/multiselect_list.jsx

```diff
@@ -178,4 +172,5 @@ MultiSelectList.propTypes = {
     onAdd: PropTypes.func,
     onSelect: PropTypes.func,
     loading: PropTypes.bool,
+    showEmail: PropTypes.bool,
```

Comment:
Similar to above, the MultiSelectList shouldn't need to know about showEmail, but it would be great if UserMultiSelectList did